_bibliography/references.bib

```diff
@@ -2,6 +2,29 @@
 ---
 references
 ==========
+@inproceedings{baser2024securespectra,
+  title={SecureSpectra: Safeguarding Digital Identity from Deep Fake Threats via Intelligent Signatures},
+  author={Baser, Oguzhan and Kale, Kaan and Chinchali, Sandeep},
+  booktitle={INTERSPEECH},
+  year={2024}
+}
+
+@inproceedings{Choi_2024_ECCV,
+  author={Choi, Minkyu and Goel, Harsh and Omama, Mohammad and Yang, Yunhao and Shah, Sahil and Chinchali, Sandeep},
+  title={Towards Neuro-Symbolic Video Understanding},
+  booktitle={Proceedings of the European Conference on Computer Vision (ECCV)},
+  month={September},
+  year={2024}
+}
+
+@inproceedings{Narayanan_2024_PEERNet,
+   author={Narayanan, Aditya and Kasibhatla, Pranav and Choi, Minkyu and Li, Po-han and Zhao, Ruihan and Chinchali, Sandeep},
+   title={PEERNet: An End-to-End Profiling Tool for Real-Time Networked Robotic Systems},
+   booktitle={IEEE/RSJ International Conference on Intelligent Robots and Systems (IROS)},
+   year={2024},
+   organization={IEEE},
+ }
+
 @article{narasimhan2024time,
```

Comment:
we should maybe decide the right naming format later. E.g., Choi, Minkyu vs. Po-han Li.

Reply:
it should be showing correctly on the website.